ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/Overview/ClustersTableContainer.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Bullseye, Divider, Spinner } from '@patternfly/react-core';
+
+import TableErrorComponent from 'Components/PatternFly/TableErrorComponent';
+import useURLPagination from 'hooks/useURLPagination';
+
+import TableEntityToolbar, { TableEntityToolbarProps } from '../../components/TableEntityToolbar';
+import { QuerySearchFilter } from '../../types';
+
+export type ClustersTableContainerProps = {
+    filterToolbar: TableEntityToolbarProps['filterToolbar'];
+    entityToggleGroup: TableEntityToolbarProps['entityToggleGroup'];
+    querySearchFilter: QuerySearchFilter;
+    isFiltered: boolean;
+    rowCount: number;
+    pagination: ReturnType<typeof useURLPagination>;
+};
+
+function ClustersTableContainer({
+    filterToolbar,
+    entityToggleGroup,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    querySearchFilter,
+    isFiltered,
+    rowCount,
+    pagination,
+}: ClustersTableContainerProps) {
+    // TODO - Placeholders for query results
+    const data = [];
+    const previousData = undefined;
+    const error: Error | undefined = undefined;
+    const loading = false;
+
+    const tableData = data ?? previousData;
+
+    return (
+        <>
+            <TableEntityToolbar
+                filterToolbar={filterToolbar}
+                entityToggleGroup={entityToggleGroup}
+                pagination={pagination}
+                tableRowCount={rowCount}
+                isFiltered={isFiltered}
+            />
+            <Divider component="div" />
+            {loading && !tableData && (
+                <Bullseye>
+                    <Spinner isSVG />
+                </Bullseye>
+            )}
+            {error && (
+                <TableErrorComponent error={error} message="Adjust your filters and try again" />
+            )}
+            {!error && tableData && (
+                <div role="region" aria-live="polite" aria-busy={loading ? 'true' : 'false'}>
```

Comment:
David, can you share this idiom at the next team meeting.

For my understanding, subsequent loading leaves table intact, and then renders data from response?

Reply:
Yes, a follow up data load would reuse the same `<div>` element and replace the contents of the table.

I tried getting a more thorough demonstration of this using MacOS VoiceOver a while back, but was unable to distinguish any difference between these aria attributes and no attributes. It might have just been due to unfamiliarity with the tool though, and is something I'd like to revisit in the future.